citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/model/impl/CrossRefPublication.java

```diff
@@ -0,0 +1,65 @@
+package edu.asu.diging.citesphere.importer.core.model.impl;
+
+import edu.asu.diging.citesphere.importer.core.model.BibEntry;
+
+public class CrossRefPublication implements BibEntry {
```

Comment:
Some of the variables like article, book chapter, etc.  are common in both, but the values. I separated the class to avoid confusion. Also, the mapping of cross ref is different than the other ones. I have used this class to identify if it's a cross ref entry. It can be merged, but tracing things might become difficult, and I don't want to break other import that is already working.